python_modules/dagster/dagster/_core/definitions/declarative_scheduling/scheduling_context.py

```diff
@@ -43,6 +46,21 @@ def get_asset_subset_updated_after_time(
             asset_key=asset_key, after_time=after_time
         )
 
+    def get_parent_asset_partitions_updated_after_child(
+        self,
+        *,
+        asset_partition: AssetKeyPartitionKey,
+        parent_asset_partitions: AbstractSet[AssetKeyPartitionKey],
+        respect_materialization_data_versions: bool,
```

Comment:
we can take off the respect_materialization_data_versions flag here -- we'll want to remove it entirely and so we can just set it to False when calling the queryer method